client/client.go

```diff
@@ -586,7 +586,7 @@ func (c *CfClient) setAnalyticsServiceClient(ctx context.Context) {
 		return
 	}
 	c.config.Logger.Info("Posting analytics data enabled")
-	c.analyticsService.Start(ctx, &c.metricsApi, c.environmentID)
```

Comment:
This was a pointer to an interface which I removed as it prevented me from passing in my own type for testing purposes.